### DISCUSSION TITLE
Some useful cryptography resourches

### DISCUSSION THREAD

**Discussion by angelospapoutsis** [OpenMined/courses#175] [Resources]
Hi,

with regard to the following

**Lesson 4: Technical Tools for Input Verification**

I made a post here https://app.slack.com/client/T6963A864/C01HY1GAG76/thread/C01HY1GAG76-1609939426.237100

presenting some resources which I think are helpful for anyone how likes to dive deeper in public-key cryptography

Helena Palmer asked me to add them here also. So I hope that it will be useful.

https://www.youtube.com/playlist?list=PLSNNzog5eyduN6o4e6AKFHekbH5-37BdV

https://www.youtube.com/playlist?list=PLSNNzog5eydtwsdT__t5WtRgvpfMzpTc7

https://dev.to/techschoolguru/a-complete-overview-of-ssl-tls-and-its-cryptographic-system-36pd

https://security.stackexchange.com/questions/7421/how-do-the-processes-for-digital-certificates-signatures-and-ssl-work

**Comment by souravcipher** [comment]
Hi @angelospapoutsis! Thanks for sharing the resources. : )
